pkg/types/machinepools.go

```diff
@@ -34,8 +34,16 @@ const (
 	ArchitectureS390X = "s390x"
 	// ArchitecturePPC64LE indicates ppc64 little endian (Power PC)
 	ArchitecturePPC64LE = "ppc64le"
+	// ArchitectureARM64 indicates arm (aarch64) systems
+	ArchitectureARM64 = "arm64"
 )
 
+// ArchitectureList contains all possible architectures
+var ArchitectureList = []Architecture{ArchitectureAMD64, ArchitectureARM64, ArchitecturePPC64LE, ArchitectureS390X}
+
+// ArchitectureListAWS contains the architectures available on AWS
+var ArchitectureListAWS = []Architecture{ArchitectureAMD64, ArchitectureARM64}
```

Comment:
It looks like we need validation to ensure that the user does not select an architecture other than amd or arm when installing to AWS.

Reply:
is this validation when the user edits install-config manually? because if created through the `create install-config` command the only options available for aws are `amd64` and `arm64`.

Reply:
added arch validation for control plane, computes and AMIs..

Reply:
Should the validation be a separate commit?